assemblies/pentaho-war/pom.xml

```diff
@@ -22,10 +22,10 @@
     <hibernate-ehcache.version>3.6.9.Final</hibernate-ehcache.version>
     <commons-io.version>2.2</commons-io.version>
     <maven-resources-plugin.version>3.0.1</maven-resources-plugin.version>
-    <tika-core.version>1.19.1</tika-core.version>
+    <tika-core.version>1.22</tika-core.version>
```

Comment:
Are we using `tika` in our code? It is a dependency of `jackrabbit` so we should let it be included as a transitive of `jackrabbit`.